c/loop-industry-pattern/aiob_1_true-unreach-call.c

```diff
@@ -125,7 +125,7 @@ typedef float  REAL32;
 UINT32 Id_MCDC_92(UINT16 Id_MCDC_88, UINT8 Id_MCDC_89, UINT32 Id_MCDC_90, void * Id_MCDC_91);
 UINT32 Id_MCDC_92(UINT16 Id_MCDC_88, UINT8 Id_MCDC_89, UINT32 Id_MCDC_93, void * Id_MCDC_91)
 {
-	UINT32 x;
+	UINT32 x=__VERIFIER_nondet_uint();
```

Comment:
You forgot to declare this nondet function as extern. See other benchmarks for an example.